LookupAnything/ModEntry.cs

```diff
@@ -122,6 +123,19 @@ private void OnGameLaunched(object? sender, GameLaunchedEventArgs e)
             monitor: this.Monitor,
             manifest: this.ModManifest
         ).Register();
+
+        // add Iconic Framework integration
+        IconicFrameworkIntegration iconicFramework = new(this.Helper.ModRegistry, this.Monitor);
+        if (iconicFramework.IsLoaded)
+        {
+            iconicFramework.AddToolbarIcon(
+                "LooseSprites/Cursors",
+                new Rectangle(330, 357, 7, 13),
+                I18n.Icon_ToggleSearch_Name,
+                I18n.Icon_ToggleSearch_Desc,
+                this.TryToggleSearch
+            );
```

Comment:
For Lookup Anything, maybe we should add a second icon to lookup something in front of the player? That could be useful for mobile players so they don't need to use a virtual keyboard.